frontend/src/components/pages/CamperRefundCancellation/CamperRefundInfoCard.tsx

```diff
@@ -0,0 +1,222 @@
+import React from "react";
+import {
+  Box,
+  Text,
+  Checkbox,
+  VStack,
+  StackDivider,
+  Flex,
+  Spacer,
+  extendTheme,
+  ChakraProvider,
+} from "@chakra-ui/react";
+import { CamperRefundDTO, RefundDTO } from "../../../types/CamperTypes";
+
+type CamperRefundInfoCardProps = {
+  camperRefund: any;
+  firstName: string;
+  lastName: string;
+  camperNum: number;
+  key: number;
+  instances: Array<CamperRefundDTO>;
+};
+
+const CamperRefundInfoCard = ({
+  camperRefund,
+  firstName,
+  lastName,
+  camperNum,
+  key,
+  instances,
+}: CamperRefundInfoCardProps): React.ReactElement => {
+  const getTimeDifference = (date1: Date, date2: Date): number => {
+    const date1Time = date1.getHours() * 60 + date1.getMinutes();
+    const date2Time = date2.getHours() * 60 + date2.getMinutes();
+
+    return date2Time - date1Time; // Difference in minutes
+  };
+
+  const getTotalEDLPTime = (instance: CamperRefundDTO): number => {
+    const start = camperRefund.startTime;
+    const end = camperRefund.endTime;
+    const [startHours, startMinutes] = start.split(":").map(Number);
+    const [endHours, endMinutes] = end.split(":").map(Number);
+
+    const startDate = new Date();
+    startDate.setHours(startHours);
+    startDate.setMinutes(startMinutes);
+
+    const endDate = new Date();
+    endDate.setHours(endHours);
+    endDate.setMinutes(endMinutes);
+
+    let total = 0;
+    const earlyDropoffs = instance.earlyDropoff;
+    const latePickups = instance.latePickup;
+
+    earlyDropoffs?.forEach((date) => {
+      total += getTimeDifference(new Date(date), startDate);
+    });
+
+    latePickups?.forEach((date) => {
+      total += getTimeDifference(endDate, new Date(date));
+    });
+    return total;
+  };
+
+  const getTotalRefundForCamper = () => {
+    let totalRefund = 0;
+    instances.forEach((instance) => {
+      totalRefund +=
+        instance.charges.earlyDropoff +
+        instance.charges.latePickup +
+        instance.charges.camp;
+    });
+    return totalRefund;
+  };
+
+  const isRefundValid = () => {
+    const dates: Date[] = [];
+
+    // get all the dates of sessions
+    instances.forEach((instance) => {
+      instance.dates.forEach((date) => {
+        dates.push(new Date(date));
+      });
+    });
+
+    // sort the dates and find the first session
+    dates.sort((a, b) => a.getTime() - b.getTime());
+    const firstDate = dates[0];
+    const thirtyDays = 30 * 24 * 60 * 60 * 1000;
+
+    return new Date().getTime() - firstDate.getTime() >= thirtyDays;
```

Comment:
believe we want to switch this to:
`return (firstData.getTime() - new Date().getTime()) >= thirtyDays`

The current implementation displays both cards as invalid refunds even though the dates are > 30 days away.